homeassistant/components/homematic/__init__.py

```diff
@@ -148,35 +148,36 @@
 DEFAULT_PASSWORD = ''
 DEFAULT_VARIABLES = False
```

Comment:
This is obsolete with the changes below. However, I would actually like to keep variables optional. At least it looks like with these changes variables are fetched by default without an option to turn that off.